pubsub.go

```diff
@@ -78,6 +78,9 @@ type PubSub struct {
 	// topics tracks which topics each of our peers are subscribed to
 	topics map[string]map[peer.ID]struct{}
 
+	// a set of notification channels for newly subscribed peers
+	newSubs map[string]chan peer.ID
```

Comment:
is this actually used?

Reply:
oops, didn't think about it bc I thought Go would flag the unused variables. My bad.